python/semantic_kernel/connectors/ai/audio_to_text_client_base.py

```diff
@@ -5,8 +5,7 @@
 from typing import Any
 
 from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
-from semantic_kernel.contents.audio_content import AudioContent
-from semantic_kernel.contents.text_content import TextContent
+from semantic_kernel.contents import AudioContent, TextContent
```

Comment:
so within the framework it is better to refer to the full path (less chance of circular imports because something else mentioned in the same __init__ gets loaded), not a big deal for something as "isolated" as contents but good to keep in mind